docker-compose.yaml

```diff
@@ -1,4 +1,20 @@
 version: '3.6'
 services:
 
-# TODO: Specify the services here. Refer cwl-wes
\ No newline at end of file
+    drs:
+        build:
+            context: .
+            dockerfile: Dockerfile
+        restart: unless-stopped
+        links: 
+            - mongodb
+        command: bash -c "cd /app/drs_filer; python app.py"
```

Comment:
TODO: Need to replace this with Gunicorn.

Reply:
We'll do Gunicorn later, it's not urgent